src/TortoiseProc/LogDataVector.cpp

```diff
@@ -64,7 +64,10 @@ int CLogDataVector::ParserFromLog(CTGitPath *path, int count, int infomask, CStr
 	CString gitrange = _T("HEAD");
 	if (range != nullptr)
 		gitrange = *range;
-	CString cmd = g_Git.GetLogCmd(gitrange, path, count, infomask, true);
+	CFilterData filter;
+	filter.m_NumberOfLogs = count;
+	filter.m_NumberOfLogsScale = CFilterData::SHOW_LAST_N_COMMITS;
+	CString cmd = g_Git.GetLogCmd(gitrange, path, infomask, true, &filter);

```

Comment:
There is only one call of this function (CLogDataVector::ParserFromLog()) which pass the parameter count = 2, otherwise count = -1.

I tested that call (count = 2) manually.
I am considering add some auto tests.